spec/features/idv/doc_auth/choose_id_type_spec.rb

```diff
@@ -192,12 +193,12 @@
     context 'desktop flow', :js do
       before do
         complete_doc_auth_steps_before_hybrid_handoff_step
+        stub_request(:get, IdentityConfig.store.dos_passport_composite_healthcheck_endpoint)
```

Comment:
This stub couldn't be done via 
```
let(:passport_api_health_check_status) { 'DOWN' }
```
since the the context is that the API goes down after the user hits the welcome page.